lib/addons/ack/index.ts

```diff
@@ -0,0 +1,108 @@
+// lib/certmanager_addon.ts
+import { ManagedPolicy } from 'aws-cdk-lib/aws-iam';
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace, setPath } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided option for the Helm Chart
+ */
+export interface AckAddOnProps extends HelmAddOnUserProps {

```

Comment:
Another issue is that this add-on will need to have a unique identifier per each service. I assume this may collapse if more than one AckAddOn is added. 
For example I need to add a dependency on IAM ACK addon,  but I have both IAM and RDS in the blueprint. How can we differentiate them? There is an ID field for each add-on for that which should be populated: https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/spi/addon-contracts.ts#L18

Reply:
Lets talk about this too. I added both `iam` and `ec2` and it worked.